heron/common/src/java/com/twitter/heron/common/basics/FileUtils.java

```diff
@@ -123,4 +123,17 @@ public static boolean isOriginalPackagePex(String packageFilename) {
   public static String getBaseName(String file) {
     return new File(file).getName();
   }
+
+  public static String getPkgType(String topologyBinaryFile) {
```

Comment:
@objmagic - hi, Is there anything I could do? The method name which @billonahill mentioned is better . However, I don't think adding an new enum class is a good idea. Perhaps we could add new static constants in FileUtils instead of handling string variable explicitly, for example

```
private static final String PEX = "pex";
```